sdk/eventhub/azure-eventhub/azure/eventhub/_producer_client.py

```diff
@@ -232,12 +232,13 @@ def create_batch(self, **kwargs):
          will assign to all partitions using round-robin.
         :keyword str partition_key: With the given partition_key, event data will be sent to
          a particular partition of the Event Hub decided by the service.
-        :keyword int max_size_in_bytes: The maximum size of bytes data that an EventDataBatch object can hold.
+        :keyword int max_size_in_bytes: The maximum size of bytes data that an EventDataBatch object can hold. By
+         default the value is determined by the Event Hubs service.
```

Comment:
...  by the Event Hubs service and your price tier

Reply:
updated, I just use the word `Event Hubs tier`